owncloudApp/src/main/java/com/owncloud/android/sharees/presentation/UsersAndGroupsSearchProvider.kt

```diff
@@ -112,27 +113,36 @@ class UsersAndGroupsSearchProvider : ContentProvider() {
         sortOrder: String?
     ): Cursor? {
         Log_OC.d(TAG, "query received in thread " + Thread.currentThread().name)
-        when (uriMatcher.match(uri)) {
-            SEARCH -> return searchForUsersOrGroups(uri)
-            else -> return null
+        return when (uriMatcher.match(uri)) {
+            SEARCH -> searchForUsersOrGroups(uri)
+            else -> null
         }
     }
 
     private fun searchForUsersOrGroups(uri: Uri): Cursor? {
         var response: MatrixCursor? = null
 
-        val userQuery = uri.lastPathSegment!!.toLowerCase()
+        val userQuery = uri.lastPathSegment!!.toLowerCase(Locale.getDefault())
 
         /// need to trust on the AccountUtils to get the current account since the query in the client side is not
         /// directly started by our code, but from SearchView implementation
         val account = AccountUtils.getCurrentOwnCloudAccount(context)
 
+        val ocCapabilityViewModel = OCCapabilityViewModel(context, account)
+        val capability = ocCapabilityViewModel.getStoredCapabilityForAccount()
+
+        val threshold = capability.filesSharingSearchMinLength ?: DEFAULT_MIN_CHARACTERS_TO_SEARCH
```

Comment:
`thresold` for what? Maybe renaming it as `searchLengthThreshold`

Reply:
Just naming, i can change it 👍